interpreter/CMakeLists.txt

```
@@ -459,7 +459,7 @@ if (builtin_clang)
   endif()
   set(CLANG_INCLUDE_DIRS
     ${CMAKE_SOURCE_DIR}/interpreter/llvm-project/clang/include
-    ${CMAKE_BINARY_DIR}/interpreter/llvm-project/llvm/tools/clang/include
+    ${CMAKE_BINARY_DIR}/interpreter/llvm-project/clang/include
```

Unfortunately, this breaks the configuration with `builtin_llvm=ON`. Looking a bit around, I think what we want is
```suggestion
    ${Clang_DIR}/include
```
This works on my system for both settings of `builtin_llvm`. Can you maybe check on your end?

---

That works too